native/cocos/bindings/jswrapper/v8/Object.cpp

```diff
@@ -170,11 +164,10 @@ Object *Object::createSetObject() {
 
 Object *Object::getObjectWithPtr(void *ptr) {
     Object *obj = nullptr;
-    auto iter = NativePtrToObjectMap::find(ptr);
-    if (iter != NativePtrToObjectMap::end()) {
-        obj = iter->second;
+    NativePtrToObjectMap::forEach(ptr, [&obj](se::Object *foundObj) {
```

Comment:
If there is more than one se::Object mapped to native pointer, which se::Object will it return?

Reply:
This method `getObjectWithPtr` seems can be removed. 

![image](https://user-images.githubusercontent.com/40414978/201315680-74d50cfe-c13c-40bf-8ea1-71f7821da6cf.png)

This is the only caller, add the returned `obj` should be identical to `seObj`

Reply:
Yes, since we store `se::Object` into JS object's internal field, it doesn't need to find se::Object through `NativePtrToSeObject` map again.

Good catch